.github/workflows/test.sh

```diff
@@ -19,8 +22,8 @@ main(){
     esac
```

Comment:
Jeg synes det er påfallende hvor likt dette er Go channels:

```
    read msg < "$done_chan"
    case "$msg" in
        TIMEOUT)
            fail "TIMEOUT" ;;
        OK)
            echo "Tests OK!"
            exit ;;
        *)
            fail "Unexpected: $msg" ;;
     esac
```

>  it can also be seen as a type-safe generalization of Unix pipes. 
> —[Effective Go - Share by communicating](https://go.dev/doc/effective_go#concurrency)